src/_data/toc/content.yml

```
@@ -29,7 +29,7 @@ pages:
     - label: Adding a New Page
       url: "/cms/page-add.html"
       children:
-      - label: Duplicating a Page
+      - label: Duplicating & Deleting a Page
```

We don't use ampersands (although the UI unfortunately has some)

```suggestion
      - label: Duplicating and Deleting a Page
```